gt_concordance/gt_concordance.py

```diff
@@ -0,0 +1,519 @@
+import sys
+import os
+import re
+import subprocess
+import shutil
+
+import click
+import yaml
+import vcf
+import pyexcel_xlsx
+
+from ngi_pipeline.database.classes import CharonSession, CharonError
+
+
+CONFIG = "/lupus/ngi/production/v1.5/conf//irma_ngi_config_sthlm.yaml"
+
+@click.group()
+@click.pass_context
+@click.option('--config', '-c', default=CONFIG, help='Path to a config file', required=True, type=click.Path())
+def cli(context, config):
+    if config == CONFIG:
+        click.echo("Using default config file: {}".format(CONFIG))
+    else:
+        click.echo('Config file: {}'.format(os.path.abspath(config)))
+    if not os.path.exists(config):
+        click.echo('Config file does not exist!')
+        exit(1)
+    with open(config, 'r') as config_file:
+        config = yaml.load(config_file) or {}
+    context.obj = config
+
+@cli.command()
+@click.pass_context
+def parse_xl_files(context):
+    config = context.obj
+    # checking config
+    XL_FILES_PATH = config.get('XL_FILES_PATH')
+    if XL_FILES_PATH is None:
+        click.echo("config file missing XL_FILES_PATH argument")
```

Comment:
this needs to be written to the standard log file we have in irma, or better, to a specific log file used to trace genotype stuff. I suggest you to use the typical logging class we use